lib/src/main/java/org/team100/lib/motion/components/VelocityServo.java

```diff
@@ -15,6 +15,12 @@ public interface VelocityServo<T extends Measure100> extends Glassy {
      * @param setpoint
      */
     void setVelocity(double setpoint);
+    /**
+     * Velocity in meters/sec or radians/sec depending on T.
+     * 
+     * @param setpoint
```

Comment:
if you use @param javadoc you need to list them all.

also you should name the accel setpoint something with the word "accel" in the name, and in the comment you should point out that the units are m/s^2 or rad/s^2